### ISSUE TITLE
EMI Create recipes break when Create:Metalwork is installed

### ISSUE THREAD

**Issue by Sylphily** [AverageAnime/create-metalwork#23]
after working it down, when i have Create:Metalwork installed it prevents me from seeing any of the create assemblies, such as tracks. This occurred on fabric version.

**Comment by Vinsard** [comment]
Yeah, I have the same problem. In mod version 1.0.8 but in version 1.0.7 this problem does not exist.
